internal/controller/spectrumxrailpoolconfig_host_flows_controller.go

```
@@ -311,7 +334,140 @@ func (r *SpectrumXRailPoolConfigHostFlowsReconciler) configureXPlane(ctx context
+	if err != nil {
+		return fmt.Errorf("failed to create X-Plane bridges", err)
+	}
+	if err := r.deployXplane(ctx, r.Client, rpc, r.Scheme, namespace, config.FromEnv()); err != nil {
+		return fmt.Errorf("failed to deploy xplane: %w", err)
+	}
+
+	return nil
+}
+
+func (r *SpectrumXRailPoolConfigHostFlowsReconciler) deployXplane(ctx context.Context, client k8sclient.Client, poolConfig *v1alpha1.SpectrumXRailPoolConfig,
+	scheme *runtime.Scheme, namespace string, cfg *config.OperatorConfig) error {
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofumpt)